dev-utils/docker-compose.yml

```diff
@@ -1,6 +1,7 @@
-version: "2.1"
+version: "2.4"
```

Comment:
seems like the version is not supported in travis, check the below error 
```
ERROR: Version in "././dev-utils/docker-compose.yml" is unsupported. You might be seeing this error because you're using the wrong Compose file version. Either specify a supported version (e.g "2.2" or "3.3") and place your service definitions under the `services` key, or omit the `version` key and place your service definitions at the root of the file to use version 1.
For more on the Compose file format versions, see https://docs.docker.com/compose/compose-file/
The command "docker-compose -f ./dev-utils/docker-compose.yml up -d wait-for-apm-server" exited with 1.


```

Reply:
I'll check if I can keep 2.1, I think I changed it to support something of the network configuration but I'm not sure

Reply:
done, now it works on Travis too.